cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -523,11 +512,11 @@ public TestEventsMessagesLinker getTestEventsMessagesLinker()
 	
 	
 	@Override
-	protected Iterable<StoredMessage> doGetMessages(StoredMessageFilter filter) throws IOException
+	protected Iterable<StoredMessage> doGetMessages(StoredMessageFilter filter, Order order) throws IOException

```

Comment:
I think it worth adding an `Order` field to `StoredMessageFilter`. This will be at least backward-compattible with previous version of Cradle API.